src/client/caching/stores/InviteStore.ts

```diff
@@ -10,4 +12,61 @@ export class InviteStore extends DataStore<Invite> {
 		super(client, extender.get('Invite'), client.options.cache.limits.invites);
 	}
 
+	/**
+	 * Returns a {@link Invite invite} with optionally their metadata.
+	 * @since 0.0.1
+	 * @param code The {@link Invite#code invite code}.
+	 * @see https://discord.com/developers/docs/resources/invite#get-invite
+	 */
+	public async fetch(code: string, options: InviteStoreFetchOptions = {}): Promise<Invite> {
+		const endpoint = Routes.invite(code);
+
+		// eslint-disable-next-line @typescript-eslint/camelcase
+		const entry = await this.client.api.get(endpoint, { query: { with_counts: true, ...options } }) as APIInviteData;
```

Comment:
Whats the point of this? You should do the default in the `= {}` in the function param